cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -1464,6 +1464,14 @@
     </description>
   </property>
 
+  <property>
+    <name>app.transaction.batch.size</name>
```

Comment:
Nit: change transaction to "streaming". We are using it both for transactions and microbatching, but it's implementation detail. Main thing it's microbatches for streaming use cases